apps/frontend/src/context/HeartRateContext.tsx

```diff
@@ -4,53 +4,32 @@ import {
   useHeartRateMonitorInterface,
 } from '../hooks/useHeartRateMonitorInterface';
 import { useHeartRateMonitorMock } from '../hooks/useHeartRateMonitorMock';
+import { useMockSettings } from './MockContext';
 
 const HeartRateContext = React.createContext<HeartRateMonitorInterface | null>(
   null
 );
 
-export const HeartRateRealContextProvider = ({
-  children,
-}: {
-  children: React.ReactNode;
-}) => {
-  const heartRateMonitor = useHeartRateMonitorInterface();
-
-  return (
-    <HeartRateContext.Provider value={heartRateMonitor}>
-      {children}
-    </HeartRateContext.Provider>
-  );
-};
-
-export const HeartRateMockContextProvider = ({
-  children,
-}: {
-  children: React.ReactNode;
-}) => {
-  const heartRateMonitor = useHeartRateMonitorMock();
-
-  return (
-    <HeartRateContext.Provider value={heartRateMonitor}>
-      {children}
-    </HeartRateContext.Provider>
-  );
-};
-
```

Comment:
Just simplified this by removing the 2 extra provider "components". Not sure if that has any cons